samples/azure-quantum/CONTRIBUTING.md

```diff
@@ -17,6 +17,9 @@ There are a few principles to keep in mind when writing samples that are going t
 1. The command to log in to Azure Quantum should be in its own cell (if applicable to the notebook).
 1. There should be no empty cells in the notebook.
 1. To run in the portal notebook experience, the notebook must be entirely self-contained; it cannot reference other files on disk.
+1. Try to keep samples short and runnable within 5 minutes. This gives users the best experience rather than starting a sample that may take hours to run (avoid qpu targets, except in suggestions for things to try next, since qpu targets can queue for a long time).
+1. Try to keep to using provider targets that are free, so that users are not unexpectedly charged or lose credits if choosing to run all cells. If using cells that consume money/credits, consider commenting out the cell or requiring some kind of user input to run against the non-free target.
+1. Put yourself in the user's shoes and try to address any points of confusion or longer wait times (such as if cell runs for 30 seconds, try to warn the user in the markdown cell before it). If cell will submit several jobs, try to note how many jobs will be submitted and add ("Submitting x of n jobs... print logs, so that they can see their current progress).
```

Comment:
`IntProgress` from `ipywidgets` is available in the notebook installation and is a useful tool to display progress when multiple jobs are processed or scheduled.

Reply:
Would be awesome if it works! I know that ipywidgets currently have some issues in Azure Notebooks, I believe (they don't update values as they should, so would be great to see if this widget works). If it does I can update the docs to suggest that!